apps/windows/package.json

```
@@ -3,11 +3,16 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
+    "build": "fluentui-scripts build",
     "bundle": "react-native rnx-bundle --dev false",
+    "just": "fluentui-scripts",
+    "clean": "fluentui-scripts clean",
+    "code-style": "fluentui-scripts code-style",
     "depcheck": "fluentui-scripts depcheck",
     "e2etest": "rimraf reports/* && wdio",
     "generate-report": "allure generate allure-results --clean && allure open",
     "lint": "eslint .",
+    "prewindows": "install-windows-test-app",
```

We should use the NuGet packages. It'll cut down the build times by _a lot_.
```suggestion
    "prewindows": "install-windows-test-app --use-nuget",
```

---

Oh, we should do the same in `.ado/templates/e2e-testing-uwp.yml`.

---

And the README

---

Since install-windows-test-app Is a `prewindows` step, the README should probably just not mention it at all. I'll remove it from the README and add it to the pipeline and here